helm/values.yaml

```diff
@@ -15,25 +15,33 @@ deployment:
   tolerations: []
   affinity: {}
 
-# configure connaisseur service
+# configure Connaisseur service
 service:
   type: ClusterIP
   port: 443
 
-# configure access to the notary server
+### VALIDATORS ###
+# validators are a set of configurations (hosts, types, public keys) that can be
+# used for validating one or multiple images (or image signatures). They are tied
+# to their respective image(s) via the image policy below. There are a few handy
+# validators pre-configured.
 validators:
+# static validator that allows each image
 - name: allow
   type: static
   approve: true
+# static validator that denies each image
 - name: deny
   type: static
   approve: false
-- name: default  # the default validator which is used, if no validators are specified
+# the `default` validator is used if no validator is specified in image policy
+- name: default
   type: notaryv1  # cosign is also experimetally supported
```

Comment:
If `cosign` is the correct type, I'd highlight it with `"cosign"` if you want to have this comment here

Reply:
I think this comment doesn't really fit here. Maybe add a list of supported validators to the comment in the beginning?

Reply:
Ah I think I know what you wanted to say: I'd change the comment to something like "this is an example only; you can choose from the list of available validators"